src/router.js

```diff
@@ -1,10 +1,15 @@
 const express = require('express');
 const path = require('path');
 
+const controllers = require('./controllers');
+
 const router = express.Router();
 
+router.post('/addActivity', controllers.activity.store);
```

Comment:
So if you change the names this would be: `router.post('/addActivity', controllers.activity.add);` which makes a bit more sense to me